test/run-verify-host-user.sh

```diff
@@ -167,6 +167,9 @@ if [ -n "$test_basic" ]; then
     if [ "$TEST_OS" = "fedora-36" ]; then
         EXCLUDES="$EXCLUDES TestSOS.testBasic"
     fi
+
+    # Exclude NFS tests due to a timing issue https://github.com/cockpit-project/cockpit/issues/17142
+    EXCLUDES="$EXCLUDES TestStorageNfs"
```

Comment:
Oh, you need to specify the full name. Unlike the opt-in "tests" list, this needs to be precise and not substrings (this would be too error prone otherwise).